weather-app/src/components/InfoBox.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Text, View, StyleSheet } from "react-native";
+import { Feather } from "@expo/vector-icons";
+
+const InfoBox = (props) => {
+  const { wind, humidity, chanceOfRain } = props;
+  return (
+    <View
+      style={[
+        styles.flexRow,
+        {
+          backgroundColor: "rgba(34, 107, 163, 0.7)",
+          borderRadius: 30,
+          borderColor: "white",
+          borderWidth: 1,
+        },
+      ]}
```

Comment:
Avoid having styles like this, extract them into styles constant. Same comment is for all components accross project so take care of that for better practices